cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -136,6 +140,12 @@ configure_kernel_module_locking() {
   fi
 
   if [ "${#sed_cmds[@]}" -gt 0 ]; then
+      # Check secure boot before try to modify kernel cmdline.
+      if is_secure_boot_enabled; then
+        echo "enabled"
```

Comment:
nit: `s/enabled/Secure boot is enabled` or something along those lines maybe?

Reply:
I just deleted the line. It's not necessary because we have the error message the next line.